src/commands/Games/higherLower.ts

```diff
@@ -0,0 +1,267 @@
+			cooldown: 7,
+			description: language => language.tget('COMMAND_HIGHERLOWER_DESCRIPTION'),
+			extendedHelp: language => language.tget('COMMAND_HIGHERLOWER_EXTENDED'),
+			requiredPermissions: ['ADD_REACTIONS', 'MANAGE_MESSAGES', 'EMBED_LINKS', 'USE_EXTERNAL_EMOJIS'],
+			runIn: ['text'],
+			usage: '<50|100|200|500|1000|2000|5000|10000>'
+		});
+	}
+
+	public async run(message: KlasaMessage, [text]: [string]) {
+		await message.author.settings.sync();
+
+		const wager = Number(text);
+		const balance = message.author.settings.get(UserSettings.Money);
+		if (balance < wager) throw message.language.tget('GAMES_NOT_ENOUGH_MONEY', balance);
+
+		const response = await message.sendLocale('COMMAND_HIGHERLOWER_LOADING');
+		const game: HigherLowerGameData = {
+			/** The game's reaction collector */
+			llrc: new LongLivingReactionCollector(this.client, reaction => {
+				if (game.callback === null) return;
+
+				const emoji = this.resolveCollectedEmoji(message, game, reaction);
+				if (emoji === null) return;
+
+				game.callback(emoji);
+				game.callback = null;
+			}, async () => {
+				if (game.callback !== null) {
+					game.callback(null);
+					game.callback = null;
+				}
+				try {
+					await this.end(game, message, game.turn > 1);
+				} catch (error) {
+					this.client.emit(Events.Wtf, error);
+				}
+			}),
+			response,
+			running: true,
+			turn: 1,
+			number: this.random(0),
+			wager,
+			emojis: this.kFirstReactionArray,
+			callback: null
+		};
+
+		while (game.running) {
+			await game.response.reactions.removeAll();
+			// Send the embed
+			const { TITLE, DESCRIPTION, FOOTER } = message.language.tget('COMMAND_HIGHERLOWER_EMBED');
+			await game.response.edit(null, new MessageEmbed()
+				.setColor(getColor(message))
+				.setTitle(TITLE(game.turn))
+				.setDescription(DESCRIPTION(game.number))
+				.setFooter(FOOTER));
+
+			// Add the options
+			if (game.turn > 1) game.emojis = this.kReactionArray;
+			for (const emoji of game.emojis) {
+				await game.response.react(emoji);
+			}
+
+			const emoji = await this.awaitReactionAction(game);
+			if (emoji === null) break;
+
+			// Main game logic (at last)
+			const oldNum = game.number;
+			game.number = this.random(game.number);
+
+			switch (emoji) {
+				case HigherLowerReactions.Higher: {
+					game.running = await (game.number > oldNum ? this.win(game, message) : this.loss(game, message));
+					break;
+				}
+				case HigherLowerReactions.Lower: {
+					game.running = await (game.number < oldNum ? this.win(game, message) : this.loss(game, message));
+					break;
+				}
+				case HigherLowerReactions.Cashout: {
+					await this.end(game, message, true);
+					game.running = false;
+					break;
+				}
+			}
+
+			if (game.running) game.turn++;
+		}
+	}
+
+	private async win(game: HigherLowerGameData, message: KlasaMessage) {
+		const { language } = message;
+		await game.response.reactions.removeAll();
+
+		// TODO(Quantum): Implement Winning event for InfluxDB
```

Comment:
```suggestion
		// TODO (Quantum): Implement Winning event for InfluxDB
```

Space for consistency with other TODO's